### ISSUE TITLE
feat: Ability to pipe-in initial question

### ISSUE THREAD

**Issue by tadq** [dustinblackman/oatmeal#15]
It would be beneficial to get initial chat started by entering multi-line text in the command line:

$ oatmeal << EOF
echo "Hello World" | wc -w
How to understand this code
EOF

**Comment by dustinblackman** [comment]
Good idea! I got a WIP in [this branch](https://github.com/dustinblackman/oatmeal/compare/stdin?expand=1), but this overall doesn't play nicely with crossterm. It'll take more effort to figure out how to do it.

**Comment by tadq** [comment]
I use `micro` as secondary text editor while VS Code as primary. 

`micro` uses Lua as scripting language. I am willing to try making oatmeal plugin for it at some point.
I am not great at Lua. I am Go/Python dev. I see that you have neovim plugin in Lua too. This might be a good start.

It might be beneficial to have pipe-in and pipe-out interface in oatmeal. Just allow pipe in initial text and spit out resulting `/a` append command back as stdout. This should make plugins in other text editors easier to write.

**Comment by dustinblackman** [comment]
Nice I'm glad you got something working!

The UI is technically writing to stdout. We wouldn't be able to output to stdout it unless the app intends to exit and close the UI first.
